riff-raff/public/docs/howto/fix-a-failed-deploy.md

```diff
@@ -13,16 +13,21 @@ the desired capacity in excess of your configured max.
 
 ## Step 1 - get the ASG back to it's normal size
 
-Use the AWS console to change the desired size of the ASG back to the original value (typically half). 
+Use the AWS console to change the desired size of the ASG back to the original value (typically half). Access the AWS console via Janus, navigate to the EC2 section, then click "Autoscaling Groups." The desired size can be modified by clicking the "edit" button in the "Details" tab. The Riff-Raff error and logs should let you know which ASGs have been used for your given deploy.
```

Comment:
Hi Jacob, sorry my reply is so delayed. I get very annoyed when referring to docs that are very presumptuous that a given user will have enough familiarity with a domain (eg. AWS/servers) to just 'get' themselves to the right place, which is why I thought it was better to spell it out. I think it would be better to say something like, if the UI navigation has changed and you can't find this, please pair with another developer and update these docs to reflect the new nav.

Reply:
Fair enough. In practice I doubt this will change particularly often anyway (and as you say, this can be fixed quite easily when there are changes) so I'm happy to go with your preference here 👍